woocommerce/woocommerce
# E2e/add build to e2e core tests

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds a build step to the build command of the `e2e-core-tests` package. It uses the pre-existing build system already in use by `e2e-environment` and `e2e-utils` to transpile the JS files using Babel.

The original build script requires the code to be transpiled to be inside a folder called `src`, so the entrypoint `index.js` was moved to `src/index.js`, the `specs` folder was moved to `src/specs`, and the `main` property of the `package.json` file now points to `build/index.js`, which is the transpiled version of `src/index.js`.

Closes #31396 .

### How to test the changes in this Pull Request:

1. Run `pnpm prepare`
2. Assert that you see two new folders: `build` and `build-modules`, and that inside of these folders you see a transpiled version of `src`.
3. Run the tests and assert they complete without errors `pnpm nx test-e2e woocommerce`

### Other information:




### Changelog entry

Added to CHANGELOG.md.

### FOR PR REVIEWER ONLY:
